packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-side-note/signup-side-note-provider.tsx

```diff
@@ -0,0 +1,157 @@
+    openWizard: VoidFunction;
+};
+
+const getNote = ({
+    country_code,
+    history,
+    initial_deposit_per_country,
+    openWizard,
+    paymentagent_details,
+}: TGetNoteProps): TNote => {
+    const can_apply_to_be_a_payment_agent = !!paymentagent_details?.can_apply;
+    const is_applied = paymentagent_details.status === 'applied';
+    const is_authorized = paymentagent_details.status === 'authorized';
+    const is_rejected = paymentagent_details.status === 'rejected';
+    const is_verified = paymentagent_details.status === 'verified';
+    const insufficient_deposit = paymentagent_details.eligibility_validation?.some(
+        status => status === 'PaymentAgentInsufficientDeposit'
+    );
+    const needs_poi = paymentagent_details.eligibility_validation?.some(status => status === 'NotAgeVerified');
+    const needs_poa = paymentagent_details.eligibility_validation?.some(status => status === 'NotAuthenticated');
+    const locked_account = paymentagent_details.eligibility_validation?.some(
+        status => status === 'PaymentAgentClientStatusNotEligible'
+    );
+    const initial_deposit =
+        initial_deposit_per_country[country_code.toUpperCase()] || initial_deposit_per_country.default;
+
+    if (insufficient_deposit) {
+        return is_rejected
+            ? {
+                  button_text: <Localize i18n_default_text='Deposit now' />,
+                  description: (
+                      <Localize
+                          i18n_default_text='If you want to become a payment agent, please fund your account with a minimun of <0/> USD to be eligible.'
+                          components={[<Money key={0} amount={initial_deposit} />]}
+                      />
+                  ),
+                  icon: 'IcAlertWarning',
+                  is_primary_button: true,
+                  onClick: () => history.push(routes.cashier_deposit),
+                  title: <Localize i18n_default_text="You don't meet the requirements" />,
+                  title_color: 'warning',
+              }
+            : {
+                  button_text: <Localize i18n_default_text='Top-up now' />,
+                  description: (
+                      <Localize
+                          i18n_default_text="We've paused your application review as you've insufficient account balance. Please top-up your account until you have a balance of <0/> USD for us to continue the review."
+                          components={[<Money key={0} amount={initial_deposit} />]}
+                      />
+                  ),
+                  icon: 'IcCashierRedWarning',
+                  is_primary_button: true,
+                  onClick: () => history.push(routes.cashier_deposit),
+                  title: <Localize i18n_default_text='Sign-up failed' />,
+                  title_color: 'loss-danger',
+              };
+    } else if (is_rejected && (needs_poi || needs_poa)) {
```

Comment:
@george-usynin-binary Since you are returning in each condition, no need to `else if`, you can replace them with `if`